### PR TITLE
Slight improvements to Dockerfile

### PR DESCRIPTION
**Pull request by jsf9k**
# 

## 🗣 Description

* Move the definition of `SCANNER_HOME` to the top of the `Dockerfile`
* Rename `SCANNER_HOME` to `HOME`
* Remove some unnecessary braces

## 💭 Motivation and Context

These changes clean up the `Dockerfile` a little bit.

## 🧪 Testing

I tested via a full scan run in my staging workspace.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -41,14 +46,15 @@ RUN pip install --no-cache-dir --upgrade pshtt==0.6.6
...
+    ${HOME}/domain-scan/
 RUN pip install --no-cache-dir --upgrade \
-    --requirement /home/scanner/domain-scan/requirements.txt
+    --requirement ${HOME}/domain-scan/requirements.txt
 
 ###
 # Remove build dependencies
 ###
-RUN apt-get remove --quiet --quiet ${BUILD_DEPS}
+RUN apt-get remove --quiet --quiet $BUILD_DEPS
```
Comment (mcdonnnj) [inline Dockerfile]: Was this supposed to be INSTALL_DEPS?

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Digging the multi-stage work 👍 Please see my comment about the variable used.

File: Dockerfile (hunk 1 above)
Comment (jsf9k) [reply Dockerfile]: Great catch!  Fixed in 86f1ea3.
:baseball:

Comment (felddy) [reply Dockerfile]: Quality!

**Review by felddy — APPROVED** [review]
💎

File: Dockerfile (hunk 1 above)
Comment (jsf9k) [reply Dockerfile]: > Quality!

[What is quality?](https://en.wikipedia.org/wiki/Pirsig%27s_Metaphysics_of_Quality)

**Review by mcdonnnj — APPROVED** [review]
It's beautiful ✨

**Review by dav3r — APPROVED** [review]
Fabuloso! ⚙️

**Pull request merged**